src/v-sticky/sticky.vue

```diff
@@ -0,0 +1,26 @@
+<script>
+
+    export default {
+        inserted (el, binding) {
+
+            const offset = binding.arg
+
+            window.addEventListener('scroll', () => {
+
+                if (!el.getBoundingClientRect) return
+
+                const rect = el.getBoundingClientRect()
```

Comment:
Super nitpicky note here 😄 , but you could do

```js
const rect = el.getBoundingClientRect()

if (!rect) return
```

so you only have to fetch the rect once